Sources/SwifterSwift/SwiftStdlib/MutableCollectionExtensions.swift

```diff
@@ -22,3 +22,19 @@ public extension MutableCollection where Self: RandomAccessCollection {
         sort { $0[keyPath: keyPath] < $1[keyPath: keyPath] }
     }
 }
+
+public extension MutableCollection  {
+    /// SwifterSwift: Assign to an given value to a field of all elements of the collection
+    ///
+    /// - Parameter value: The new value of the field
+    /// - Parameter keyPath: The actual field of the element
+    mutating func assignToAll<Value>(value: Value, by keyPath: WritableKeyPath<Element,Value>) {
```

Comment:
I'm not sure I find this any clearer or succinct than:

`collection.forEach { $0[keyPath: path] = value }`

Reply:
I'm new to this, working on this issue. https://github.com/SwifterSwift/SwifterSwift/issues/759.
I agree that your version is more succinct

Reply:
@chulbert 
I was doing a code review where I saw a lot of those `views.forEach { $0.isHidden = true }`, so I thought this would be nice. 
I'm not sure it is more succinct or better, maybe not ... I just thought it was nice and would give someone here a good first issue :))

Reply:
> I'm not sure I find this any clearer or more succinct than:
> 
> `collection.forEach { $0[keyPath: path] = value }`
> 
> Can you elaborate on the use case you intend to serve?

I tried forEach and it didn't work as the $0 become immutable there